### PR TITLE
api: Surface a new attribute instructions that contains OCIL

### PR DESCRIPTION
**Pull request by jhrozek**
Extends the ComplianceCheckResult API object with a new attribute
instructions that contains the OCIL attribute from OpenScap rule.

This attribute contains the manual checks for cases when a rule cannot
be evaluated with a OVAL check.

### CODE REVIEW

File: deploy/crds/compliance.openshift.io_compliancecheckresults_crd.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ spec:
           id:
             description: A unique identifier of a check
             type: string
+          instructions:
+            description: Manual check steps in case the rule can't be evaluated automatically
```
Comment (JAORMX) [inline deploy/crds/compliance.openshift.io_compliancecheckresults_crd.yaml:39]: Does this only show if there rule can't be evaluated automatically?

**Comment by jhrozek** [comment]
On Thu, Feb 11, 2021 at 09:47:04AM -0800, Juan Osorio Robles wrote:
> @JAORMX commented on this pull request.
> 
> 
> 
> > @@ -35,6 +35,9 @@ spec:
>            id:
>              description: A unique identifier of a check
>              type: string
> +          instructions:
> +            description: Manual check steps in case the rule can't be evaluated automatically
> 
> Does this only show if there rule can't be evaluated automatically? 

No, always.

**Comment by JAORMX** [comment]
> On Thu, Feb 11, 2021 at 09:47:04AM -0800, Juan Osorio Robles wrote:
> > @JAORMX commented on this pull request.
> > 
> > 
> > 
> > > @@ -35,6 +35,9 @@ spec:
> >            id:
> >              description: A unique identifier of a check
> >              type: string
> > +          instructions:
> > +            description: Manual check steps in case the rule can't be evaluated automatically
> > 
> > Does this only show if there rule can't be evaluated automatically? 
> 
> No, always.
> 

Let's change the description, with the current way it's written out sounds like that key will only have text if automatic remediation wasn't possible

**Pull request merged**